commit 864993a65c5576d702cd2e3d840d6cba55be2102
Author: Alexandre Bailon <abailon@baylibre.com>
Date:   Wed Feb 1 21:30:12 2017 -0600

    usb: musb: da8xx: Remove CPPI 3.0 quirk and methods
    
    commit a994ce2d7e66008381a0b184c73be9ae9b72eb5c upstream.
    
    DA8xx driver is registering and using the CPPI 3.0 DMA controller but
    actually, the DA8xx has a CPPI 4.1 DMA controller.
    Remove the CPPI 3.0 quirk and methods.
    
    Fixes: f8e9f34f80a2 ("usb: musb: Fix up DMA related macros")
    Fixes: 7f6283ed6fe8 ("usb: musb: Set up function pointers for DMA")
    Signed-off-by: Alexandre Bailon <abailon@baylibre.com>
    Acked-by: Sergei Shtylyov <sergei.shtylyov@cogentembedded.com>
    Acked-by: Tony Lindgren <tony@atomide.com>
    Signed-off-by: Bin Liu <b-liu@ti.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1308_864993a65c5576d702cd2e3d840d6cba55be2102.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1308_864993a65c5576d702cd2e3d840d6cba55be2102.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
